docs/app-configuration.md

```diff
@@ -45,6 +45,17 @@ ARG BUCKET_NAME=${BUCKET_NAME}
 ARG SERVER_ENDPOINT=${SERVER_ENDPOINT}
 ```
 
+As well as the variables you set yourself, CapRover will also set a `CAPROVER_GIT_COMMIT_SHA` environment variable to the full git commit SHA that is being deployed. This is only available during the Docker build and is not available inside your app by default. If you want to use it inside your app then you can use something like the following:
+
+```
+FROM imagename....
+ARG CAPROVER_GIT_COMMIT_SHA=${CAPROVER_GIT_COMMIT_SHA}
+
+RUN echo $CAPROVER_GIT_COMMIT_SHA > ./CAPROVER_GIT_COMMIT_SHA
```

Comment:
There is an easier way to use ARG as an ENV:
```
ENV CAPROVER_GIT_COMMIT_SHA=${CAPROVER_GIT_COMMIT_SHA}
```

Reply:
I've updated the PR to do it that way. I wasn't 100% sure if the `ARG` was still needed (fairly new to Docker) but I think it is so I've left it in. Let me know if that's not correct.

Reply:
Looks good, thanks!